lib/puppet/provider/clickhouse.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Parrent class for ClickHouse providers
+class Puppet::Provider::Clickhouse < Puppet::Provider
+  desc 'This provides basic clickhouse-client to execute server side commands
+
+To use custom clickhouse-client settings create /root/.clickhouse-client/config.xml'
+
+  initvars
+
+  ENV['PATH'] += ':/bin:/usr/bin:/usr/local/bin'
```

Comment:
i would not do that. the env should be user-provided and its the users responsibility to set it up correctly no?

Reply:
Hmm, most probably, I misread the mysql provider a little bit https://github.com/puppetlabs/puppetlabs-mysql/blob/master/lib/puppet/provider/mysql.rb#L7

Will change it to append, but not override it. Good catch

Reply:
Ah, but it is `+=` already. And without this part, AFAIR, it breaks somehow. I'd leave it as is now.

Reply:
ok fine. i was just saying that the user might not want to use those paths :)